src/amo/components/SearchForm/index.js

```diff
@@ -27,6 +27,7 @@ import Icon from 'ui/components/Icon';
 
 import './styles.scss';
 
+export const SEARCH_TERM_MIN_LENGTH = 2;
```

Comment:
If we already have a constant for max length (which I didn't even know about/remember!) I think it's fine to have one for minimum length too. I guess it's more extensible if they're props with defaults set using `static defaultProps = { autocompleteMinLength: 3 };`

But it's not a huge deal either way.